src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/Manipulation_Tests.cs

```diff
@@ -23,5 +26,72 @@ public void TestManipulation()
 
 			TakeScreenshot("Result");
 		}
+
+		[Test]
+		[AutoRetry]
+		[ActivePlatforms(Platform.Android, Platform.iOS)] // PinchToZoomInCoordinates is not supported on WASM yet
```

Comment:
Open an issue for that one

Reply:
https://github.com/unoplatform/Uno.UITest/issues/27